packages/axios-extension/test/abap/abap-service-provider.test.ts

```diff
@@ -76,10 +76,10 @@ describe('AbapServiceProvider', () => {
     });
 
     // Discovery schema is cached, so separate this test suite from other ADT service tests
-    describe('Service unavailable in discovery', () => {
+    describe.only('ADT Services unavailable in discovery', () => {
```

Comment:
```suggestion
    describe('ADT Services unavailable in discovery', () => {
```

Reply:
@ullasholla already fixed. Noticed it when the sonar qgate dropped.